### PR TITLE
elasticsearch: update to 5.2.1

### PR DESCRIPTION
**Pull request by ayghor**
EHLO!

could someone review this?

some points you may yell at me for:

 * split into elasticsearch5 instead of updateing elasticsearch because it may break stuff
 * kinda followed teh directory structure from teh debian package

im in a hurry these days but will try to fix asap.

### CODE REVIEW

**Comment by ebfe** [comment]
Unless someone complains i don't think we need to keep the old version.

**Comment by ebfe** [comment]
cc @bougyman

**Comment by ayghor** [comment]
i will apply over teh old pkg then!

**Comment by ayghor** [comment]
wat bout this?

File: srcpkgs/elasticsearch/template [hunk 1]
```diff
@@ -1,41 +1,48 @@
...
-version=1.7.3
-revision=2
+version=5.1.2
+revision=1
+wrksrc="elasticsearch-${version}"
 only_for_archs="i686 x86_64"
 depends="virtual?java-runtime"
 short_desc="A distributed, open source search and analytics engine"
-maintainer="bougyman <bougyman@voidlinux.eu>"
+maintainer="dota? =op <rogi@skylittlesystem.org>"
```
Comment (Gottox) [inline srcpkgs/elasticsearch/template]: I'm excited to see if that will break something😆

Comment (ayghor) [inline srcpkgs/elasticsearch/template]: lint didnt complain its perfectly safe

Comment (ayghor) [inline srcpkgs/elasticsearch/template]: =O3

**Pull request merged**